docs/README-develop-beaker-scripts.md

```diff
@@ -121,7 +60,7 @@ HOSTS:
     <Nexus agent>:
         roles:
             - agent
-        platform: cisco-5-x86_64
+        platform: cisco_nexus-7-x86_64
```

Comment:
It's the OS major version (NX-OS 7.x, IOS XR 6.x). Not what I would have preferred but it's what the Beaker folks decided on.